cookbook/service_container/event_listener.rst

```diff
@@ -71,53 +77,153 @@ using a special "tag":
 
     .. code-block:: yaml
 
-        # app/config/services.yml
+        # app/config/config.yml
         services:
             kernel.listener.your_listener_name:
-                class: AppBundle\EventListener\AcmeExceptionListener
+                class: AppBundle\Listener\ExceptionListener
                 tags:
-                    - { name: kernel.event_listener, event: kernel.exception, method: onKernelException }
+                    - { name: kernel.event_listener, event: kernel.exception }
 
     .. code-block:: xml
 
-        <!-- app/config/services.xml -->
-        <service id="kernel.listener.your_listener_name" class="AppBundle\EventListener\AcmeExceptionListener">
-            <tag name="kernel.event_listener" event="kernel.exception" method="onKernelException" />
+        <!-- app/config/config.xml -->
+        <service id="kernel.listener.your_listener_name" class="AppBundle\Listener\ExceptionListener">
+            <tag name="kernel.event_listener" event="kernel.exception" />
         </service>
 
     .. code-block:: php
 
-        // app/config/services.php
+        // app/config/config.php
         $container
-            ->register('kernel.listener.your_listener_name', 'AppBundle\EventListener\AcmeExceptionListener')
-            ->addTag('kernel.event_listener', array('event' => 'kernel.exception', 'method' => 'onKernelException'))
+            ->register('kernel.listener.your_listener_name', 'AppBundle\Listener\ExceptionListener')
+            ->addTag('kernel.event_listener', array('event' => 'kernel.exception'))
         ;
 
 .. note::
 
-    There is an additional tag option ``priority`` that is optional and defaults
-    to 0. The listeners will be executed in the order of their priority (highest to lowest).
-    This is useful when you need to guarantee that one listener is executed before another.
+    There is an optional tag option called ``method`` which defines which method
+    to execute when the event is triggered. By default the name of the method is
+    ``on`` + "camel-cased event name". If the event is ``kernel.exception`` the
+    method executed by default is ``onKernelException()``.
+
+    The other optional tag option is called  ``priority`` and it defaults to ``0``.
+    This value ranges from ``-255`` to ``255`` and it controls the order in which
```

Comment:
Are there really any limits besides the range of the `int` type? IIRC, these are only the minimum and maximum priorities used in the core.